motiontoast/src/main/java/www/sanju/motiontoast/MotionToast.kt

```diff
@@ -151,7 +152,7 @@ class MotionToast {
                             successToastColor
                         )
                     )
-                    layout.custom_toast_text.text = TOAST_SUCCESS
+                    layout.custom_toast_text.text = if(title == null) TOAST_SUCCESS else title
```

Comment:
Hey @NotNotMarshall, Thanks for making a PR. It looks good to me. It will be better if we use this approach.

Instead of checking  `title == null`, we can use the extension function `title.isNullOrBlank()`

**Sample code:**

` layout.custom_toast_text.text = if(title.isNullOrBlank()) TOAST_SUCCESS else title`


Can please make a change in all states like this?

Reply:
Done mate! ^_^